build-support/publish-zip-to-artifactory.sh

```diff
@@ -9,6 +9,5 @@ ZIP=${PARENT}/target/universal/streamliner-$VERSION.zip
 test -f $ZIP
 
 echo "Uploading ${ZIP}"
-cloudsmith push raw phdata/streamliner --version $VERSION $ZIP
```

Comment:
Publishing our ZIP to a MAVEN repo is working just fine, no need to use RAW as well.